app/static/js/categories_filter.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener("DOMContentLoaded", function () {
+  document.getElementById("category-button").addEventListener("click", function () {
+    filterByCategory();
+  });
+});
+
+function filterByCategory(){
+    // TODO(issue#67): Allow filter by category name
+    const category = document.getElementById("category").value;
+
+    const css = (!Number.isInteger(+category) || !category || 0 === category.length) ? `
+    .event_line {
+      display: grid;
+      visibility: visible;
+    }
+    ` : `
+    .event_line {
+      display: none;
+      visibility: hidden;
+    }
+    .event_line[data-name*="${category}" i] {
+      display: grid;
+      visibility: visible;
+    }
+    `;
```

Comment:
It's usually considered a bad practice to directly edit the CSS.
Maybe you can toggle a class, or a data attribute, and use `.event-line[data-active="on"]` and `.event-line[data-active="off"]`?

Reply:
I can change the CSS part and instead change in js (something similar to the loops below...), WDYT?

Reply:
Usually we want the JavaScript to change the value of the `data-` attributes in the HTML, or toggle the classes of the elements in the DOM.
It's better to avoid dynamically editing the CSS using JavaScript, if possible.

Is there a way to move this section of code to the CSS, and to change only the `data-` attributes/classes of the relevant elements?